internal/component/prometheus/relabel/relabel.go

```diff
@@ -85,6 +88,8 @@ type Component struct {
 	exited           atomic.Bool
 	ls               labelstore.LabelStore
 
+	debuggingStreamHandler livedebugging.DebugStreamManager
```

Comment:
Is it a "stream handler" or a "stream manager" or is it a "service"? 
I am definitely against adding too many concepts, especially when it's not needed. Maybe everything should be called service? 

Feel free to split it however you like in the `internal` structure of the `service/livedebugging` package, but the API exposed to Alloy components etc. should be minimalistic, consistent and properly named.